<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Plot</class>
 <widget class="QDialog" name="Plot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>График</string>
  </property>

  <widget class="QLabel" name="expression_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>360</width>
     <height>60</height>
    </rect>
   </property>
  </widget>

  <widget class="QCustomPlot" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>360</width>
     <height>360</height>
    </rect>
   </property>
  </widget>

  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>X START</string>
   </property>
  </widget>

  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Y START</string>
   </property>
  </widget>

  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>430</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>X END</string>
   </property>
  </widget>

  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>470</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Y END</string>
   </property>
  </widget>

  <widget class="QDoubleSpinBox" name="lineEdit_x1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>430</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
  </widget>

  <widget class="QDoubleSpinBox" name="lineEdit_y1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>470</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
  </widget>

  <widget class="QDoubleSpinBox" name="lineEdit_x2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>430</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
  </widget>

  <widget class="QDoubleSpinBox" name="lineEdit_y2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>470</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_replot">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>430</y>
     <width>80</width>
     <height>60</height>
    </rect>
   </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
   <property name="text">
    <string>Обновить</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
